totalRP3/Core/UITools.lua

```diff
@@ -373,43 +373,19 @@ end
 -- Tooltip tools
 --*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
 
-TRP3_API.ui.tooltip.CONFIG_TOOLTIP_SIZE = "CONFIG_TOOLTIP_SIZE";
-local CONFIG_TOOLTIP_SIZE = TRP3_API.ui.tooltip.CONFIG_TOOLTIP_SIZE;
 local getConfigValue = function() end;
 
-TRP3_API.RegisterCallback(TRP3_Addon, TRP3_Addon.Events.WORKFLOW_ON_LOADED, function()
-	TRP3_API.configuration.registerConfigKey(TRP3_API.ui.tooltip.CONFIG_TOOLTIP_SIZE, 11);
-	getConfigValue = TRP3_API.configuration.getValue;
-end);
-
-local function getTooltipSize()
-	return getConfigValue(CONFIG_TOOLTIP_SIZE) or 11;
-end
-
 -- Show the tooltip for this Frame (the frame must have been set up with setTooltipForFrame).
 -- If already shown, the tooltip text will be refreshed.
 local function refreshTooltip(Frame)
 	if Frame.titleText and Frame.GenFrame and Frame.GenFrameX and Frame.GenFrameY and Frame.GenFrameAnch then
-		TRP3_MainTooltip:Hide();
-		TRP3_MainTooltip:SetOwner(Frame.GenFrame, Frame.GenFrameAnch,Frame.GenFrameX,Frame.GenFrameY);
-		if not Frame.rightText then
-			TRP3_MainTooltip:AddLine(Frame.titleText, 1, 1, 1, true);
-			TRP3_TooltipUtil.SetLineFontOptions(TRP3_MainTooltip, 1, getTooltipSize() + 4);
-		else
-			TRP3_MainTooltip:AddDoubleLine(Frame.titleText, Frame.rightText);
-			TRP3_TooltipUtil.SetLineFontOptions(TRP3_MainTooltip, 1, getTooltipSize() + 4);
-			TRP3_MainTooltipTextRight1:SetNonSpaceWrap(true);
-			TRP3_MainTooltipTextRight1:SetTextColor(1, 1, 1);
-		end
-		do
-			TRP3_MainTooltipTextLeft1:SetNonSpaceWrap(true);
-			TRP3_MainTooltipTextLeft1:SetTextColor(1, 1, 1);
-		end
+		TRP3_MainTooltip:SetOwner(Frame.GenFrame, Frame.GenFrameAnch, Frame.GenFrameX, Frame.GenFrameY);
+		GameTooltip_SetTitle(TRP3_MainTooltip, Frame.titleText);
```

Comment:
There was no mention of rightText removal but there doesn't seem to be any case where it was used so makes sense

Reply:
Oh right - yeah, I'd noticed it was entirely unused.